primeqa/mrc/run_mrc_utils.py

```diff
@@ -32,4 +35,167 @@ def _split_into_class_and_module_name(class_path):
             return getattr(module_reference, object_name)
     except Exception as ex:
         traceback.print_exc()  # Shows additional traceback for why imports fail
-        raise TypeError(f"Unable to resolve the string {reference_as_str} to a fully qualified class path") from ex
\ No newline at end of file
+        raise TypeError(f"Unable to resolve the string {reference_as_str} to a fully qualified class path") from ex
+
+
+def get_raw_datasets(fof, data_args, task_args, cache_dir, split='train'):
```

Comment:
Why doesn't the default HF get_raw_datasets work?

Reply:
@seirasto I can't find the function from HF transformers. The function is extended from the one in run_mrc.py to let it process datasets with different formats, and was moved to run_mrc_utils.py to make the run_mrc.py cleaner.